spec/acceptance/organizer/reduce_if_spec.rb

```diff
@@ -49,6 +49,13 @@ def self.actions
     expect(result).to be_success
   end
 
+  it "knows that it's being reduced from within an organizer" do
+    result = TestDoubles::TestIterate.call(:number => 1,
+                                           :counters => [1, 2, 3, 4])
+
+    expect(result.organized_by).to eq TestDoubles::TestIterate
```

Comment:
Did you want to test it with `reduce_if`?

Reply:
Yes - I wanted to make sure all `reduce_if`, `reduce_until`, and `iterate` "meta-organizers" were handled. I've amended the expectation text to make their test cases a little clearer.

Reply:
I meant you used `TestIterate`, when the file itself is `reduce_if_spec`. I'll merge it, but will fix it.